lib/cdo/safe_browsing.rb

```diff
@@ -36,8 +36,22 @@ def self.determine_safe_to_open(url_to_check)
       response = http.start {http.request(req)}
     end
 
+    threat_type = "Determined safe"
+
     response_value = response.nil? ? nil : JSON.parse(response.body).empty?
```

Comment:
I find it confusing that `response_value` is a boolean, not the actual response value. Would it be clearer to just use the parsed response body?
```
response_body = response.nil? ? nil : JSON.parse(response.body)
if response_body.nil?
      threat_type = 'No response from API'
    elsif !response_body.empty?
...
response_body.nil? ? true : (error_response || response_body.empty?)
```